Purchases/Attribution/AttributionFetcher.swift

```diff
@@ -43,16 +43,7 @@ class AttributionFetcher {
     }
 
     var identifierForVendor: String? {
-        // Should match available platforms in
-        // https://developer.apple.com/documentation/uikit/uidevice?language=swift
-        // https://developer.apple.com/documentation/watchkit/wkinterfacedevice?language=swift
```

Comment:
I think it would be a good idea to add this comment into the implementation of the `var identifierForVendor: String?` property in `SystemInfo`.

Reply:
Yup 👍